src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -48,10 +52,6 @@ function CreateReminder() {
     userId,
     ...toDoData
   };
```

Comment:
You should change the last version of the todoData. ref.current?.value is used at the final stage before submission. You should think here. IMPORTANT part is here